recipe/meta.yaml

```diff
@@ -61,7 +61,6 @@ test:
     - test ! -f $PREFIX/lib/libGLESv2${SHLIB_EXT}              # [linux]
     - test ! -f $PREFIX/lib/libEGL${SHLIB_EXT}                 # [linux]
     - test ! -f $PREFIX/lib/libGL${SHLIB_EXT}                  # [linux]
-    - test -f $PREFIX/lib/libOSMesa${SHLIB_EXT}                # [linux]
```

Comment:
Upstream dropped libosmesa

Reply:
there is another libOSMesa check below just for osx